webapi/Skills/ChatSkills/ExternalInformationSkill.cs

```diff
@@ -94,7 +98,7 @@ public class ExternalInformationSkill
             return string.Empty;
         }
 
-        var contextString = string.Join("\n", context.Variables.Select(v => $"{v.Key}: {v.Value}"));
+        var contextString = string.Join("\n", context.Variables.Where(v => !(v.Key.Contains("TokenUsage", StringComparison.CurrentCultureIgnoreCase) || v.Key.Contains("tokenLimit", StringComparison.CurrentCultureIgnoreCase) || v.Key.Contains("chatId", StringComparison.CurrentCultureIgnoreCase))).Select(v => $"{v.Key}: {v.Value}"));
```

Comment:
nit: maybe worth putting this in a method with better description of what this does and why this has to be done

Reply:
Good call, moved this out to new ` private string GetChatContextString(SKContext context)` method